### ISSUE TITLE
I don't have an issue title currently

### ISSUE THREAD

**Issue by suprstarrd** [jbmagination/Luma3DS#14]
> 
> 
> @T0biasCZe
> 
> Just a note in regards to the usage of the filter.
> 
> You'll need to go into the screen filter settings and set persistence to On and set your desired color temperature, save the settings, and then reboot.
> 
> On any fresh boot you have to manually set the color temperature once, or sleep and resume. After that your filter settings will be re-applied on wake.

@vertexmachina Hello, what system do you tested it on ? I tried it, i set the colour, set the persistance to on, and saved and rebooted it, and sometimes, after opening/closing it loads back to menu with the blue filter applied, and sometimes its stucked on black screen
I have new 2ds xl

_Originally posted by @T0biasCZe in https://github.com/jbmagination/Luma3DS/issues/5#issuecomment-621366992_

**Comment by vertexmachina** [comment]
I use a New 2DS XL and haven't experienced any issues. Unfortunately I don't know what the problem could be.

**Comment by suprstarrd** [comment]
Yeah @T0biasCZe Sorry man, it’s more of an experimental feature and is not supported by AW/Luma. It was just added back as a toggle feature, if you don’t like it then you can always just disable persistence

**Comment by hide0123** [comment]
> after opening/closing it loads back to menu with the blue filter applied, and sometimes its stucked on black screen

Putting the unit to sleep before the home menu is completely loaded may result in black screen. So wait a few seconds for the home menu to be fully loaded.

**Comment by suprstarrd** [comment]
I tried it upon first open and had the issue
